docs/advanced/input_files/input-main.md

```diff
@@ -711,26 +711,26 @@ calculations.
 
 - **Type**: String
 - **Description**: Choose the basis set.
-  - pw: Using plane-wave basis set only.
-  - lcao_in_pw: Expand the localized atomic set in plane-wave basis.
-  - lcao: Using localized atomic orbital sets.
+  - **pw**: Using plane-wave basis set only.
+  - **lcao_in_pw**: Expand the localized atomic set in plane-wave basis.
```

Comment:
It is unable in current version, you'd better delete this option in documentation in this PR.

Reply:
There are still many "lcao_in_pw" in codes, I suggest to delete them in a new pr, and then delete this option.

Reply:
> There are still many "lcao_in_pw" in codes, I suggest to delete them in a new pr, and then delete this option.

This option is not useless, but can not be used in this version, it can be discuss whether fix or delete.

Reply:
Is it OK to write as "- **lcao_in_pw**: (Unavailable now) Expand the localized atomic set in plane-wave basis"?

Reply:
> Is it OK to write as "- **lcao_in_pw**: (Unavailable now) Expand the localized atomic set in plane-wave basis"?

Okay, maybe "Unavailable currently, it will be fixed in future versions".

Reply:
OK